### ISSUE TITLE
Hoobs unable find device in HOME app 

### ISSUE THREAD

**Issue by weber267** [tuya/tuya-homebridge#312]
No error shows, but can't find any device. 
![Screen Shot 2022-10-22 at 10 25 39 PM](https://user-images.githubusercontent.com/68871162/197370150-ee467ec8-d2f6-492b-9e1e-5870e9d91285.png)

**Comment by 0x5e** [comment]
Please try install [@0x5e/homebridge-tuya-platform](https://github.com/0x5e/homebridge-tuya-platform) beta version, and enable homebridge debug mode. We will see which step the issue happened.

**Comment by DavKirsch** [comment]
I had this same issue and just solved it today. About a month ago, all my devices on the Tuya Platform bridge stopped working in Homekit. They were still working on the Smart Life app though...

After a lot of searching, I discovered the IoT Core Services on the Tuya IoT Developer Platform had been working via a free trial and that had expired. There was an orange button to request an extension of the trial (as buying the bundle was $25K+). I asked for an extension for 6 months for my Hoobs Smart Hub, which was granted by the next morning. Then, I restarted the Bridge in HOOBs and all my devices were cached again and working in Homekit!! Took me a month to solve but everything is up and running.

FYI, You will need to repeat this process every 6 months though, the only downside.

[Workflow.with.Tuya.pdf](https://github.com/tuya/tuya-homebridge/files/10296130/Workflow.with.Tuya.pdf)
